controller/workitem.go

```diff
@@ -66,6 +67,17 @@ func (c *WorkitemController) List(ctx *app.ListWorkitemContext) error {
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError("could not parse filter", err))
 	}
+	if ctx.FilterExpression != nil {
+		q := *ctx.FilterExpression
+		// Better approach would be to convert string to Query instance itself.
+		// Then add new AND clause with spaceID as another child of input query
+		// Then convert new Query object into simple string
+		queryWithSpaceID := fmt.Sprintf(`{"%s":[{"space": "%s" }, %s]}`, search.Q_AND, ctx.SpaceID, q)
```

Comment:
@hectorj2f are you talking about `ctx.FilterExpressoion` or `ctx.FilterAssignee` and such `if` blocks?
Else, I did not understand your comment, can you please elaborate more?
thanks.